README.md

```diff
@@ -242,9 +242,9 @@ Otherwise, the time is when the route should arrive to the destination by
 
 Time is in epoch (seconds since 1/1/1970) |
 
-*required* **end** : [{Double, Double}] - “[{< latitude1 Double >,< longitude1 Double >}]”
+*required* **end** : [String] - “[“< latitude Double >,< longitude Double >”]”
```

Comment:
can we check with iOS, because this should optimally be an array of float tuples –– I really want to convert this to a POST request, so that we get better encoding of booleans and floats, and we don't have to do this manually on the server cc @ywsang